src/core/include/iDynTree/Core/EigenHelpers.h

```diff
@@ -386,6 +386,15 @@ inline void setSubVector(VectorDynSize& vec,
     return;
 }
 
+inline void setSubVector(VectorDynSize& vec,
+                         const IndexRange range,
+                         const LinearMotionVector3& linMotionVector)
+{
+    assert(range.size==3);
+    toEigen(vec).segment(range.offset, range.size) = toEigen(linMotionVector);
+    return;
+}
```

Comment:
Is this necessary? Now `LinearMotionVector3` is just a `GeomVector3`, that inherits from `VectorFixSize<3>, so the previous setSubVector overload should apply also here. Anyhow, if this is necessary I guess we should set it to `GeomVector3`, as `LinearMotionVector3` is deprecated.